plugins/builds/types/models.d.ts

```diff
@@ -0,0 +1,476 @@
+/**
+ * TODO In the future, this file will be moved to 'screwdriver-models'
+ */
+// -----------------------------------------------------------------------------
+//      screwdriver-models.PipelineModel
+// -----------------------------------------------------------------------------
+/**
+ * PipelineModel is defined in 'screwdriver-models/lib/pipeline.js'
+ *
+ * @description import {PipelineModel} from 'screwdriver-models/lib/pipeline'
+ */
+export type PipelineModel = {
+    id: number;
+    name: string;
+    scmUri: string;
+    scmContext: string;
+    scmRepo: ScmRepo;
+    createTime: string;
+    admins: Record<string, boolean>,
+    workflowGraph: WorkflowGraph;
+    annotations: {};
+    lastEventId: number;
+    configPipelineId: undefined;
+    childPipelines: undefined;
+    prChain: boolean;
+    parameters: {};
+    settings: {};
+    state: string;
+    subscribedScmUrlsWithActions: Array<unknown>
+    badges: undefined;
+    templateVersionId: undefined;
+
+    // symbol
+    '[symbol(model)]': undefined;
+    '[symbol(table)]': undefined;
+    '[symbol(datastore)]': undefined;
+    '[symbol(scm)]': ScmRouter;
+    '[symbol(multiBuildClusterEnabled)]': boolean;
+    '[symbol(row data)]': undefined;
+    '[symbol(dirty variable)]': Array<string>;
+    // functions
+    getConfiguration: () => unknown;
+    addWebhooks: () => unknown;
+    syncPRs: () => unknown;
+    syncPR: () => unknown;
+    sync: () => unknown;
+    admin: Promise<UserModel>
+    tokens: Promise<unknown>
+    getFirstAdmin: () => unknown;
+    getFirstRepoAdmin: () => unknown;
+    token: Promise<unknown>
+    branch: Promise<unknown>
+    rootDir: Promise<unknown>
+    openPullRequests: Promise<unknown>
+    pipelineJobs: Promise<unknown>
+    pullRequestJobs: Promise<unknown>
+    secrets: Promise<unknown>
+    configPipeline: () => unknown;
+    getJobs: () => unknown;
+    getEvents: () => unknown;
+    getBuilds: () => unknown;
+    update: () => unknown;
+    remove: () => unknown;
+    getMetrics: () => unknown;
+    chainPR: () => boolean;
+};
+
+// -----------------------------------------------------------------------------
+//      screwdriver-models.Job
+// -----------------------------------------------------------------------------
+/**
+ * Job is defined in 'screwdriver-models/lib/job.js'
+ *
+ * @description import {Job} from 'screwdriver-models/lib/job'
+ */
+export type Job = {
+    // property
+    title: undefined;
+    createTime: undefined;
+    username: undefined;
+    userProfile: undefined;
+    url: undefined;
+    id: number;
+    name: string;
+    prParentJobId: undefined;
+    permutations: Array<Permutation>;
+    description: undefined;
+    pipelineId: number;
+    state: string;
+    stateChanger: undefined;
+    stateChangeTime: undefined;
+    stateChangeMessage: undefined;
+    archived: boolean;
+    templateId: undefined;
```

Comment:
are most of these meant to be `undefined`?

Reply:
Sorry, these `.d.ts` files were already unnecessary :bow:

We used these files to understand models.
Thus some of them didn't have correct property's type.

I cleaned up these files and fixed JSDocs.